src/test/coll.rs

```diff
@@ -753,3 +756,25 @@ async fn no_read_preference_to_standalone() {
 
     assert!(!command_started.command.contains_key("$readPreference"));
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+#[function_name::named]
+async fn watch_test() {
+    let _guard = LOCK.run_concurrently().await;
+
+    let client = EventClient::new().await;
+    if client.is_standalone() {
+        return;
+    }
+
+    let coll = client.database(function_name!()).collection("coll");
```

Comment:
I think this will still error on 3.6 if the collection doesn't exist; I think it needs to be explicitly created. Also, to ensure we don't invoke `create_collection` with an already existing collection, be sure to still use `init_db_and_coll` which drops it first. Lastly, we typically use the function name for both the database and the collection.

Reply:
We still need to invoke `create_collection` here otherwise this test will continue to fail ([example from latest run of this PR](https://evergreen.mongodb.com/task_log_raw/mongo_rust_driver_stable_tests__async_runtime~async_std_auth_and_tls~noauth_and_notls_os~macos_10.14_test_3.6_sharded_cluster_patch_3ca7a68a50022f20592cc44175bd3938bab8a117_5f249f370ae6060c2dc2d803_20_07_31_22_46_16/0?type=T#L1262))

Reply:
fixed